kano_updater/commands/install.py

```diff
@@ -225,7 +225,10 @@ def install_pip_packages(progress):
 
         success = run_pip_command("install --upgrade {}".format(pkg))
 
-        # TODO: abort the install?
         if not success:
             msg = "Installing the '{}' pip package failed".format(pkg)
             logger.error(msg)
+            if not is_internet():
+                msg="Network is down, aborting PIP install"
+                logger.error(msg)
+                raise IOError(msg)
```

Comment:
@Ealdwulf You make an interesting point. You are right, sometimes we do depend on the pip packages. The majority of the time these are just updates to those packages rather than adding new ones but you're right, we might be in a situation where we add a pip package and depend on it - maybe we should abort install.